packages/api-bindings/src/graphql/common.graphql

```diff
@@ -22,25 +22,51 @@ fragment ModuleFeeAmount on ModuleFeeAmount {
   value
 }
 
-fragment ReferenceModule on ReferenceModule {
-  ... on FollowOnlyReferenceModuleSettings {
-    __typename
-    contractAddress
+fragment AaveFeeCollectModuleSettings on AaveFeeCollectModuleSettings {
```

Comment:
I see you already included new fragments from my other PR here 😄

Reply:
Yes, I had to cause it was generating types with `| {}` otherwise. I will address the merge conflict if need to.